src/Invoice_GenUI/Invoice_GenUI/Models/Services/BaseService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+
+namespace Invoice_GenUI.Models.Services
+{
+    public class BaseService
+    {
+        public HttpClient CreateHttpClient()
+        {
+            var client = new HttpClient();
+            client.BaseAddress = new Uri("https://2023-invoice-gen.azurewebsites.net/");
+            return client;
+        }
+        public async Task<T?> SendHttpRequest<T>(string url) // T can be nullable
```

Comment:
:wrench: this should be named `SendHttpGetRequest` as it more correctly communicates what the method does.